src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/move_eltwise_up_data_movement.cpp

```diff
@@ -52,19 +53,22 @@ ov::intel_cpu::MoveEltwiseUpThroughDataMov::MoveEltwiseUpThroughDataMov() {
             return false;
         }
 
-        if (eltwise->get_output_size() == 0 ||
-            eltwise->get_input_size() == 0 ||
-            eltwise->get_output_element_type(0) != eltwise->get_input_element_type(0) ||
-            eltwise->get_output_target_inputs(0).size() != 1) {
+        if (eltwise->get_output_target_inputs(0).size() != 1) {
             return false;
         }
 
-        bool is_binary_op = std::dynamic_pointer_cast<ov::op::util::BinaryElementwiseArithmetic>(eltwise) != nullptr;
-        if (is_binary_op && !is_scalar_like(eltwise->get_input_node_shared_ptr(1))) {
-            return false;
+        for (size_t i = 1; i < eltwise->get_input_size(); ++i) {
+            if (!is_scalar_like(eltwise->get_input_node_shared_ptr(i))) {
+                return false;
+            }
         }
 
-
+        const auto fq = ov::as_type_ptr<ov::op::v0::FakeQuantize>(eltwise);
+        if (!fq) {
+            if (eltwise->get_output_element_type(0) != eltwise->get_input_element_type(0)) {
+                return false;
+            }
+        }
```

Comment:
Minor: we can simplify the code a bit:
```suggestion
        if (!ov::is_type<ov::op::v0::FakeQuantize>(eltwise) && eltwise->get_output_element_type(0) != eltwise->get_input_element_type(0)) {
            return false;
        }
```

Reply:
Done.